tests/storage/intermediate/test_storage_manager.py

```diff
@@ -21,6 +21,13 @@ def tearDown(self):
         graphene_store = GrapheneStore()
         graphene_store.remove_test_datafiles()
 
+    def assertIsNoneOrEOF(self, item):
+        """
+        Since values at the end of a file are still nonexistent, both None and
```

Comment:
We should refactor this into a GeneralTest class that includes the TearDown and SetUp we do for everything. Good refactoring.